docs/Antora/modules/howto_guides/pages/howto_read_with_domain.adoc

```diff
@@ -142,7 +142,12 @@ Cpp::
 +
 [source,cpp]
 ----
-auto reader = StreamReader<double, Int>(signal);
```

Comment:
I'm just thinking that we should probably change all of our reader factories with signals to have an optional parameter for "SkipEvents". This would make the examples a lot simpler.

Reply:
Should I implement is a part of this PR?